daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -35,10 +35,25 @@ const serverHost = "127.0.0.1"; // App Host of this Example Server
 const serverPort = "50051"; // App Port of this Example Server
 
 // HTTP Example
-const server = new DaprServer(serverHost, serverPort, daprHost, daprPort);
+const server = new DaprServer({
+  serverHost,
+  serverPort,
+  clientOptions: {
+    daprHost,
+    daprPort,
+  },
+});
 
 // GRPC Example
-const server = new DaprServer(serverHost, serverPort, daprHost, daprPort, CommunicationProtocolEnum.GRPC);
+const server = new DaprServer({
+  serverHost,
+  serverPort,
+  communicationProtocol: CommunicationProtocolEnum.GRPC,
```

Comment:
```suggestion
  communicationProtocol: CommunicationProtocolEnum.GRPC, // If DaprClient does not have a communication protocol specified, it will use the same protocol as DaprServer.
```

Reply:
Actually, from a user perspective, it may take him sometime to understand that why both Server options and Client options have communication protocol separately. So, a different section may help him understand this at once.

Reply:
It feels too verbose as a reader, finding information in documentation is already hard. I will vote to remove it, but would like to check with @XavierGeerinck as well.

Reply:
I am fine moving it to some other place as well, but would prefer as a user some place which can show me the significance of having two different communication protocols for two different paths.
But, yeah, let me know what you guys decide as @dapr/maintainers-js-sdk and I can do accordingly.

Reply:
It is indeed too verbose I think, I would remove it for now. Let's work on docs to ensure we more clearly explain difference between server and client and the options they have

Reply:
> I need to add this somewhere at least to explain the difference between both communication protocol being different now

Indeed, we can add a comment as I proposed on top "// If DaprClient does not have a communication protocol specified, it will use the same protocol as DaprServer.", to clearly mention the behavior.

Reply:
That doesn't explain explicitly the significance of difference in communication between "Dapr to App" and "App to Dapr". And, which refers to what.
As a user, I may need to go inside source file and confirm with the comments on top of communication protocol.
But, I think I have had enough, explaining it. 
I will do what you suggested.
Users can later try digging-in to understand what refers to which side of communication.

Reply:
> That doesn't explain explicitly the significance of difference in communication between "Dapr to App" and "App to Dapr".

That is explained here already, right? https://docs.dapr.io/developing-applications/sdks/js/

Users should already know that DaprClient is used for App to Dapr, and DaprServer is used for Dapr to App.

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/22132944/225921347-f7f526c6-c7cf-431d-82f2-e8972fc93fd8.png">

Reply:
Yeah, this information already seems to be there in Dapr docs, as you show here. 
And, that's something I struggled with in my initial days with Dapr as well. There was too much information, but at times I wasn't able to connect dots as information was either not at correct place or was scattered around.
It didn't hurt to point this information here as well in js-sdk-docs as well, at least to some extent.
But, nevermind - let's not discuss it any further.
I will just do what you guys suggest in next few hours.